.github/workflows/_terraform.yml

```diff
@@ -29,7 +29,8 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y cloud-init
           sudo cloud-init schema --config-file terraform/modules/google-cloud/apps/relay/templates/cloud-init.yaml
-          sudo cloud-init schema --config-file terraform/modules/google-cloud/apps/elixir/templates/cloud-init.yaml
+          # This doesn't work if the file contains interpolated variables
```

Comment:
@bmanifold This is the issue. Fine with just not running this check for the elixir cloud-init? I did some research online but there wasn't a great way to accommodate this.

We'd have to render the template to a file first as a terraform output var, then validate it.

Reply:
Yeah, should be fine as we hopefully won't be updating the cloud-init too much